install/serverless-logic-install-kn-workflow-plugin-cli.adoc

```diff
@@ -0,0 +1,77 @@
+:_content-type: ASSEMBLY
+[id="serverless-logic-install-kn-workflow-plugin-cli"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin
+
+OpenShift Serverless Logic provides a plugin named `kn-workflow` for the Knative CLI, enabling you to set up a local workflow project using the command line.
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) CLI.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+
+.Procedure
+
+//Include the binary file once the developer provides the download link for it.
+. Download the latest `kn-workflow` binary file.
+
+. Install the `kn workflow` command as a plug-in of the Knative CLI by using the following steps:
+
+.. Copy the `kn-workflow` binary file to a directory in your PATH, such as `/usr/local/bin`, and ensure the file name is `kn-workflow`:
++
+[source,terminal]
+----
+$ cp path/to/downloaded/kn-workflow /usr/local/bin/kn-workflow
+----
+
+.. Make the binary file executable as follows:
++
+[source,terminal]
+----
+$ chmod +x /usr/local/bin/kn-workflow
+----
+
+.Verification
+. Run the following command to verify that the `kn-workflow` plugin is installed successfully:
++
+[source,terminal]
+----
+$ kn plugin list
+----
++
+After installing the plug-in, you can use `kn workflow` to run the related subcommands.
+
+. Use the workflow subcommand in Knative CLI as follows:
++
+.Aliases to use workflow subcommand
+[source,terminal]
+----
+kn-workflow
+----
++
```

Comment:
Do you need to list this alias (it's in the output below)?

Reply:
Yes